cmd/kubeadm/app/phases/addons/addons.go

```diff
@@ -0,0 +1,26 @@
+package addons
+
+import (
+	clientset "k8s.io/client-go/kubernetes"
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
+	dnsaddon "k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns"
+	proxyaddon "k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/proxy"
+)
+
+// EnsureAllAddons install all addons to a Kubernetes cluster.
+func EnsureAllAddons(cfg *kubeadmapi.MasterConfiguration, client clientset.Interface) error {
```

Comment:
please move this function client-side to be consistent; i.e. in `cmd/phases/addons.go`